sass/components/_modal.scss

```diff
@@ -315,8 +316,7 @@
         .btn {
             &.btn-primary {
                 position: relative;
-                margin: -4px 33px 0 0;
-                float: right;
+                margin-top: -4px;
```

Comment:
aligning items inside flex containers would make these tiny adjustments irrelevant ... maybe try `align-items: center` on the container?

Reply:
I tried that, but it's still out of place vertically. The `.modal-header` (as seen above) has uneven top and bottom padding, so that's probably the cause of it, but I'm worried that changing that will introduce issues in other modals because of how many places that is used. I'd rather leave this part as-is for now since it's not part of this bug